static/css/configuration.less

```diff
@@ -90,3 +90,12 @@ dl.horizontal {
     }
   }
 }
+
+.mb-50 {
```

Comment:
The danger with naming classes like this is if we decide to reduce the padding to 45px you have to change the class name everywhere. Can we use `.upgrade-config legend { margin-top: 50px }` instead? I haven't tried this so it may need some work...

Reply:
Ok, sure. I added the margin to <legend>.